contrib/check-meson-install-tags.sh

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env bash
```

Comment:
i'd much rather this was in python, and called from precommit. If it's not included in the commit checks then it's going to be very easy to commit new meson changes without install tags.

Reply:
> The script does require a configured build dir for meson introspect to work

I think we can hardcode `./venv/build` -- most people seem to use ./contrib/setup these days. If that directory isn't available then I think it's fine to skip the test.

Reply:
> I think we can hardcode ./venv/build -- most people seem to use ./contrib/setup these days. If that directory isn't available then I think it's fine to skip the test.

Yeah that's totally fine.  Even the people that don't use this the pre-commit CI job does and it will catch it (if all works as intended!).